File: lib/chars/chars_fn.py
import codecs
import time


def chars(content):
	length = len(content)
	pos = 0
	while(pos < length):
		char = content[pos]
		pos = pos + 1

f = codecs.open('page.html', encoding='utf-8')

content = f.read()

start = time.time()
chars(content)

#chars(content)

end = time.time();

print end - start

File: lib/chars/chars_fn.js
var fs = require("fs");
var content = fs.readFileSync("page.html", {encoding: "utf-8"});
console.log(content.length)
function chars(content){
	var length = content.length;
	var pos = 0;
	while(pos ++ < length){
		var chr = content[pos - 1];
	}
}
var start = Date.now();
chars(content);
var end = Date.now();
console.log(end - start);
